k8s/models/common.py

```diff
@@ -50,6 +50,20 @@ class ObjectMeta(Model):
     selfLink = ReadOnlyField(six.text_type)
     uid = ReadOnlyField(six.text_type)
 
+    
+class ObjectReference(Model):
+    apiVersion = Field(six.text_type)
+    fieldPath = Field(six.text_type)
```

Comment:
Codacy found an issue: [variable 'fieldPath' in class scope should not be mixedCase (N815)](https://app.codacy.com/manual/fiaas/k8s/pullRequest?prid=5919885)